### ISSUE TITLE
Handle sigmaa and diff map after the files are processed by target loader

### ISSUE THREAD

**Issue by boriskovar-m2ms** [m2ms/fragalysis-frontend#1166]
Handle sigmaa and diff map after the files are processed by target loader. And also event map (.ccp4). Currently the densities are disabled in the UI. It's better to fix event maps together with sigmaa and diff maps together because all of them are handled at the same places and currently I can't test them anyway because of the issue #1162

**Comment by phraenquex** [comment]
Needs B/E endpoints still.  And possibly update the density modal (if quick)

**Comment by kaliif** [comment]
What are sigmaa and diff map files? There's no reference to these either in database fields or in meta_aligner.yaml. Did they get renamed? I have these ccp4 files in SiteObservation model, any of these look familiar?
- event_map
- 2Fo-Fc_map
- Fo-Fc_map

**Comment by phraenquex** [comment]
They got "renamed" - historically, different "words" have got embedded in the field for the same thing.  When I selected those database terms, I selected the less technical/jargonny terms (though obviously they'll always be jargon) - but I neglected to remind @ConorFWild and @tdudgeon of this.  Which is why you're getting the same data with different names.

1.
* event_map = event_map (no historical jargon there)
* 2Fo-Fc = sigmaa
* Fo-Fc = diff

2. 
@ConorFWild / @tdudgeon for consistency, ideally XCA would write out the map files with the "right" names (sigmaa, diff).  Is that a big code change?

**Comment by kaliif** [comment]
@boriskovar-m2ms all map files are available in api/site_observation:
- xmap_2fofc_file
- xmap_fofc_file
- event_file

**Comment by phraenquex** [comment]
Sorry @kaliif I should have been clearer:  the maps need be called by the names they have in the database (sigmaa, diff, event).

They definitely should have a consistent naming scheme.  So if you think "xmap" is useful, then: 
* xmap_sigmaa_file
* xmap_diff_file
* xmap_event_file

@tdudgeon I didn't see an answer to my question:  can you make XCA write out the maps wit the sigmaa/diff/event names?

**Comment by tdudgeon** [comment]
@phraenquex Yes, the names of the files can definitely be changed, though this is done by Conor's code. I can dig around in the code and see if I can find the place, but might need to defer to @ConorFWild for this if it's defined in his ligand_neighbourhood_alignment repo.

Just for clarity, this is what I think we would end up with:

* **XX01ZVNS2B-x0051_B_301_XX01ZVNS2B-x0772+B+301_event.ccp4** -  unchanged
* **XX01ZVNS2B-x0051_B_301_XX01ZVNS2B-x0772+B+301_diff.ccp4** - was XX01ZVNS2B-x0051_B_301_XX01ZVNS2B-x0772+B+301_FoFc.ccp4
* **XX01ZVNS2B-x0051_B_301_XX01ZVNS2B-x0772+B+301_sigmaa.ccp4** - was XX01ZVNS2B-x0051_B_301_XX01ZVNS2B-x0772+B+301_2FoFc.ccp4

**Comment by phraenquex** [comment]
Thanks @tdudgeon - please do that in your code.  @ConorFWild can fix it in the ligand_neighbourhood_alignment repo when he's back.

Yes, those are the right names.

**Comment by tdudgeon** [comment]
The changes need to be done by @ConorFWild as they are in the ligand_neighbourhood_alignment repo

**Comment by phraenquex** [comment]
You can just rename the files in your repo, right?

**Comment by phraenquex** [comment]
@ConorFWild - these files need to be properly named.  (sigmaa, diff, event).   Executive decision.  (Justification in comments above.)

@kaliif - you have to update the target loader once Conor has done.

**Comment by phraenquex** [comment]
@tdudgeon please have a poke whether you can work out if it's a simple code fix.

@Waztom assign to Tim rights on the repo.

**Comment by tdudgeon** [comment]
I have updated the names of the map files and the keys used in the YAML.
The changes are:
https://github.com/xchem/ligand_neighbourhood_alignment/commit/121b028a33bb89d00d300e5e77f4be670db1173e
https://github.com/xchem/xchem-align/commit/a712a814913ac7c30c2a7420587941e70b593fc4

@ConorFWild I just changed the value of the constants in LNA. Let me know if there would be any unexpected side effects (failing tests ...).

The output YAML now looks like this:

```
    aligned_files:
      B:
        '301':
          XX01ZVNS2B-x0875+B+301: {structure: upload_1/aligned_files/XX01ZVNS2B-x0425/XX01ZVNS2B-x0425_B_301_XX01ZVNS2B-x0875+B+301.pdb,
            event_map: upload_1/aligned_files/XX01ZVNS2B-x0425/XX01ZVNS2B-x0425_B_301_XX01ZVNS2B-x0875+B+301_event.ccp4,
            sigmaa_map: upload_1/aligned_files/XX01ZVNS2B-x0425/XX01ZVNS2B-x0425_B_301_XX01ZVNS2B-x0875+B+301_sigmaa.ccp4,
            diff_map: upload_1/aligned_files/XX01ZVNS2B-x0425/XX01ZVNS2B-x0425_B_301_XX01ZVNS2B-x0875+B+301_diff.ccp4,
```

@kaliif You will now need to adjust the target loader to accommodate these new file names and YAML keys.
I'll send you some data to work with.
